netman.py

```diff
@@ -192,6 +192,32 @@ def SetAddress4 (self, device, ipaddr, netmask, gateway):
         rc = call(["systemctl", "restart", "systemd-networkd.service"])
         return rc
 
+    @dbus.service.method(DBUS_NAME, "s", "s")
+    def GetAddressType (self, device):
+        if not self._isvaliddev (device) : raise ValueError, "Invalid Device"
```

Comment:
this will result in a HTTP 5xx.  someday we should figure out if we want to re-use the protocol internal errors like
org.freedesktop.DBus.Error.InvalidArgs or come up with our own.